eng/pipelines/runtime.yml

```diff
@@ -321,6 +327,8 @@ jobs:
     platforms:
     - Linux_arm
     - Linux_musl_arm64
+    - iOS_arm
```

Comment:
Does this use mono or coreclr System.Private.CoreLib to build? It feels to me like this should be on its own -template entry and specify RuntimeFlavor: mono. This is lying to what the comment above states.

Reply:
This should use MonoVM SPC

Reply:
So then this should be moved to a separate -template entry and specify runtimeFlavor: mono so that it is clear what it is doing, IMHO.